src/openassetio-core/hostApi/ManagerFactory.cpp

```diff
@@ -66,6 +74,69 @@ ManagerPtr ManagerFactory::createManagerForInterface(
       managerImplementationFactory->instantiate(identifier),
       managerApi::HostSession::make(managerApi::Host::make(hostInterface), logger));
 }
+
+ManagerPtr ManagerFactory::defaultManagerForInterface(
+    const HostInterfacePtr& hostInterface,
+    const ManagerImplementationFactoryInterfacePtr& managerImplementationFactory,
+    const log::LoggerInterfacePtr& logger) {
+  const char* configPath = std::getenv(kDefaultManagerConfigEnvVarName.c_str());
+
+  if (!configPath) {
+    Str msg = kDefaultManagerConfigEnvVarName + " not set, unable to instantiate default manager.";
```

Comment:
[consider][minor] Micro-optimisation - declare `msg` at the top-level and reuse, rather than re-declaring it.

Reply:
Decided to leave as is as it is infrequently called, and this helps encapsulate each use.